packages/mars/src/options/checks.ts

```diff
@@ -31,10 +32,9 @@ export function ensureBoolean(value: unknown, message: string): asserts value is
   ensure((value) => typeof value === 'boolean', value, message)
 }
 
-const NETWORKS = ['development', 'kovan', 'ropsten', 'goerli', 'rinkeby', 'mainnet']
 const URL_REGEX = /^https?:\/\/[^\s]+$/
 function isProperNetwork(value: unknown) {
-  return typeof value === 'string' && (NETWORKS.includes(value) || URL_REGEX.test(value))
+  return typeof value === 'string' && (value in chains || URL_REGEX.test(value))
```

Comment:
Imagine such an easter egg. We rename `value` to `alice` and we have `alice in chains` 😛